### PR TITLE
css, modal representation and save-recipe-login fixes

### PR DESCRIPTION
**Pull request by guyschwartzberg-student**
- added some changes to scraper to accomodate periodic scraper

### CODE REVIEW

**Comment by LeshemNoa** [comment]
Requires this small fix 
image

File: client/src/components/RecipeList/components/RecipeCard/RecipeCard.jsx [hunk 1]
```diff
@@ -82,6 +82,8 @@ class RecipeCard extends React.Component {
       if (isLoggedIn) {
         this.setState({ loadRecipeBookChange: true });
         addRecipe(token, id);
+      } else {
+        message.info('To save recipe, please log in', 5);
```
Comment (hagaishapira) [inline client/src/components/RecipeList/components/RecipeCard/RecipeCard.jsx:86]: *a recipe

File: client/src/components/RecipeList/components/RecipeCard/RecipeCard.jsx [hunk 2]
```diff
@@ -90,6 +92,8 @@ class RecipeCard extends React.Component {
       if (isLoggedIn) {
         this.setState({ loadRecipeBookChange: true });
         removeRecipe(token, id);
+      } else {
+        message.info('To save recipe, please log in', 5);
```
Comment (hagaishapira) [inline client/src/components/RecipeList/components/RecipeCard/RecipeCard.jsx:96]: *a recipe

**Comment by guyschwartzberg-student** [comment]
@LeshemNoa still waiting for approval

**Comment by guyschwartzberg-student** [comment]
image @LeshemNoa

**Pull request merged**